zkvm/src/vm.rs

```diff
@@ -204,11 +204,11 @@ where
                 Instruction::Drop => self.drop()?,
                 Instruction::Dup(i) => self.dup(i)?,
                 Instruction::Roll(i) => self.roll(i)?,
-                Instruction::Const => unimplemented!(),
-                Instruction::Var => unimplemented!(),
+                Instruction::Const => self.const_instr()?,
```

Comment:
would it be better or awful to use raw identifier syntax?

```suggestion
                Instruction::Const => self.r#const()?,
```